apps/admin/frontend/src/components/smartcard_modal/smartcard_modal.test.tsx

```diff
@@ -171,6 +180,14 @@ test('Smartcard modal displays card details when no election definition on machi
     shouldElectionDefinitionPromptBeDisplayed: boolean;
     expectedFooter: string;
   }> = [
+    {
+      programmedUser: mockVendorUser(),
+      expectedHeading: 'Vendor Card',
+      expectedElectionString: undefined,
+      shouldResetCardPinButtonBeDisplayed: false,
+      shouldElectionDefinitionPromptBeDisplayed: true,
```

Comment:
why is the election definition prompt displayed for vendor cards?

Reply:
Ah yeah, no good reason. I initially thought that it was fine for that message to get displayed when there's no election definition, even if the card you just inserted is a vendor card. But on second thought, that could be confusing when you can't actually program vendor cards on a VxAdmin. Updated accordingly: https://github.com/votingworks/vxsuite/pull/4827/commits/625d03859143d168c73d79d01ba213c08a4446ed